README.md

```diff
@@ -7,55 +7,56 @@ These are guest names I gathered in the fall that I plan to reach out to. If you
 Copy the snippet below, add your details and submit a PR!
 
 ```
-| Full Name | @twitterHandle | Previous Career | ❌ | ❌ |
+| Full Name | [social-media-handle](url-to-social-media-profile)* | Previous Career | ❌ | ❌ |
 ```
+*if you prefer not to share your contact info publicly you can email me directly at [secondcareerdevs@gmail.com](mailto:secondcareerdevs@gmail.com)
```

Comment:
Let's remove the asterisk here as well. Capitalize "if", slap a period on the end and let's just make this a sentence.